packages/remix-dev/cli/commands.ts

```diff
@@ -234,3 +236,43 @@ export async function codemod(
     throw error;
   }
 }
+
+export async function upgrade({
+  tag,
+  packageManager,
+  projectDir = process.cwd(),
+}: {
+  tag: string;
+  packageManager: PackageManager;
+  projectDir?: string;
+}) {
+  console.log(`Checking for @remix-run package updates for ${tag}...`);
+
+  let check = await ncu({
+    cwd: projectDir,
+    target() {
+      return tag;
+    },
+    upgrade: true,
+    packageManager,
+    filter(name) {
+      return name.startsWith("@remix-run/") || name === "remix";
```

Comment:
Love this new command!  I have a local script I use for this.  One thing I ran into was excluding the router in case that's a dependency since it's not synced with the remix releases.  Here's the filter in my script:

```
        (k.startsWith("@remix-run/") || k === "remix") &&
        k !== "@remix-run/router"

```